WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController+QuickActions.swift

```diff
@@ -0,0 +1,41 @@
+import UIKit
+
+extension BlogDetailsViewController {
+    @objc func quickActionsSectionViewModel() -> BlogDetailsSection {
+        return BlogDetailsSection(title: nil,
+                                  rows: [BlogDetailsRow()],
+                                  footerTitle: nil,
+                                  category: .quickAction)
+    }
+}
+
+@objc class QuickActionsCell: UITableViewCell {
+    private var actionRow: ActionRow!
+
+    func configure(with items: [ActionRow.Item]) {
+        guard actionRow == nil else {
+            return
+        }
+
+        actionRow = ActionRow(items: items)
+        contentView.addSubview(actionRow)
+
+        setupConstraints()
+    }
+
+    private func setupConstraints() {
+        actionRow.translatesAutoresizingMaskIntoConstraints = false
+
+        let widthConstraint = actionRow.widthAnchor.constraint(equalToConstant: 350)
```

Comment:
I think we can use a constant here. 

The max button width that was being used in BlogDetailsHeaderView is 390 -- I think we should match this value.
```
        static let maxButtonWidth: CGFloat = 390
```

Reply:
Fixed in 9781f26a9af41748464157f018d63d96574bbea5